dev-tools/packer/xgo-scripts/before_build.sh

```diff
@@ -18,6 +18,26 @@ PREFIX=/build
 mkdir -p $PREFIX/homedir
 make install-home HOME_PREFIX=$PREFIX/homedir
 
+# Compile the import_dahsboard binary for the requested targets.
+if [ -d $BEAT_PATH/../libbeat/ ]; then
```

Comment:
Yeah, not very happy with this. The other way I considered was to add it [here](https://github.com/elastic/beats/blob/master/dev-tools/packer/docker/xgo-image/base/build.sh#L35), but that didn't seem much better. Perhaps if we find ourselves needing this in another place next.